driver/mobile/keyboard.go

```diff
@@ -0,0 +1,26 @@
+package mobile
+
+import (
+	"fyne.io/fyne"
+)
+
+// Keyboard represents a standard fyne keyboard
+type Keyboard int32
+
+const (
+	// DefaultKeyboard is the keyboard with default input style and "Done" return key
+	DefaultKeyboard Keyboard = iota
+	// MultiLineKeyboard is the keyboard with default input style and "return" return key
```

Comment:
I wonder if multiline should be the default? Just a question for discussion I guess.
In the current implementation we show the carriage return key I think.

Reply:
I think one line entries are more common. Also, before this changes "Done" return key was default.

Reply:
Only on some phones. Older Android and iOS phones are all on return keys by default (that was related to the previous keyboard bug)

Reply:
The struggle I have with the single line "Done" default is that you don't know what the right action is. "Done", "Go", "Search" etc are all possibilities and I'd rather not pick the wrong one if we can avoid it.